### DISCUSSION TITLE
Fruitcake\Cors\CorsService issue on Laravel upgrading from 8 to 9.5

### DISCUSSION THREAD

**Discussion by podrabinek** [laravel/framework#41600] [General]
Hello,

I've just upgraded Laravel from 8.5 to 9.5.1, everything went smoothly, but I've found one inconsistency:

On https://github.com/fruitcake/laravel-cors page there's a text about Laravel 9.2+:

_Since Laravel 9.2, this Middleware is included in laravel/framework. You can use the provided middleware, which should be compatible with the Middleware and config provided in this package. See https://github.com/laravel/laravel/pull/5825/files for the changes._

According to https://github.com/laravel/laravel/pull/5825/files I've made the changes to my composer.json:

(-)  `\Fruitcake\Cors\HandleCors::class,`
(+)  `\Illuminate\Http\Middleware\HandleCors::class,`

Then upgraded the Laravel.

Now, if we go to the `vendor/laravel/framework/src/Illuminate/Http/Middleware/HandleCors.php` file, we still see the FruitCake implementation:

```php
namespace Illuminate\Http\Middleware;

use Closure;
use Fruitcake\Cors\CorsService;
use Illuminate\Contracts\Container\Container;
use Illuminate\Http\Request;

...

    /**
     * Create a new middleware instance.
     *
     * @param  \Illuminate\Contracts\Container\Container  $container
     * @param  \Fruitcake\Cors\CorsService  $cors
     * @return void
     */
    public function __construct(Container $container, CorsService $cors)
    {
        $this->container = $container;
        $this->cors = $cors;
    }

```

I've noticed that because of VSCode intelephense highlight - "Undefined type 'Fruitcake\Cors\CorsService'."

What should I do with this? Thanks

**Comment by volkankaban** [comment]
Leave it alone, latest framework upgrade still using Fruitcake\Cors\CorsService;

https://github.com/laravel/framework/blob/9.x/src/Illuminate/Http/Middleware/HandleCors.php

<?php

namespace Illuminate\Http\Middleware;

use Closure;
use Fruitcake\Cors\CorsService;
use Illuminate\Contracts\Container\Container;
use Illuminate\Http\Request;
